tracer/test/benchmarks/Benchmarks.Trace/DuckTyping/DuckTypeMethodCallComparisonBenchmark.cs

```diff
@@ -0,0 +1,366 @@
+using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using Datadog.Trace.BenchmarkDotNet;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.ClrProfiler.CallTarget.Handlers;
+using Datadog.Trace.DuckTyping;
+using static Benchmarks.Trace.DuckTyping.ObscureObject;
+
+namespace Benchmarks.Trace.DuckTyping
+{
+    [DatadogExporter]
+    [MemoryDiagnoser]
+    [GroupBenchmarksBy(BenchmarkLogicalGroupRule.ByCategory)]
+    [CategoriesColumn]
+    public class DuckTypeMethodCallComparisonBenchmark
```

Comment:
These are good to have - my only question is whether we want to run them all the time 🤔 There's a lot here, so they probably add a while to the benchmark run time?

Reply:
Currently all DuckType benchmarks are being ignored.
https://github.com/DataDog/dd-trace-dotnet/blob/53c005ddb1925f765fda61879d73c00c95b87fab/tracer/test/benchmarks/Benchmarks.Trace/Benchmarks.Trace.csproj#L22-L29

Reply:
Oh yeah 🤦‍♂️ Nice 👍